src/main/scala/org/constellation/primitives/PeerManager.scala

```diff
@@ -215,74 +214,53 @@ case class Deregistration(ip: String, port: Int, id: Id)
 case object GetPeerInfo
 
 case class UpdatePeerInfo(peerData: PeerData)
+case class ChangePeerState(id: Id, state: NodeState)
+
+case class PeerNotification(id: Id,
+                            state: PeerState,
+                            timestamp: LocalDateTime = LocalDateTime.now())
+    extends Signable
+
+object PeerState extends Enumeration {
+  type PeerState = Value
+  val LEAVE, JOIN = Value
```

Comment:
I think it should be `PascalCase` to keep consistency with other enumerations in the project